jwedb/tests/test_edb_interface.py

```diff
@@ -47,7 +67,7 @@ def test_invalid_query():
     end_time = Time('2019-01-16 00:01:00.000', format='iso')
     try:
         query_single_mnemonic(mnemonic_identifier, start_time, end_time, token='1234')
-    except RuntimeError:
+    except ValueError:
         pass
```

Comment:
In my experience, a more robust way of making sure that a certain error is raised is using `with pytest.raises` like so:
```
with pytest.raises(ValueError) as error_message:
    ~ code to test~

assert "some string to test" in str(error_message)
```
Then, the test will fail if the specific expected error isn't raised. Let me know if that example isn't clear, though!

Reply:
Thanks! This is much better.